powerauth-java-server/src/test/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/OperationServiceBehaviorTest.java

```diff
@@ -89,8 +89,27 @@ void testCreateOperationWithActivationId() throws Exception {
 
         final OperationDetailResponse operationDetailResponse = operationService.createOperation(request);
         final OperationEntity savedEntity = operationRepository.findOperation(operationDetailResponse.getId()).get();
+
+        assertTrue(operationRepository.findOperation(operationDetailResponse.getId()).isPresent());
+        assertEquals(ACTIVATION_ID, savedEntity.getActivationId());
+        assertNull(operationDetailResponse.getProximityOtp());
```

Comment:
Assert to null is true IFF the template itself is not parameterized for PAC. This is a little bit strange. but overall acceptable.

Reply:
🤔 true, I will add implicit and explicit proximity check enables.